app/src/androidTest/java/com/example/myapplication/BikeRentalApiInstrumentedTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.myapplication
```

Comment:
If these tests are duplicates of the other's that don't require a live emulator/device, let's get rid of them. I have a feeling that these tests will drift out of sync with the main library tests and cause future headaches and confusion as to why they exist, as they won't be run regularly on CI.

Reply:
Makes sense. I was going to remove them before we merged this because they're useless as of now.